# HG changeset patch
# User Rik <rdrider0-list@yahoo.com>
# Date 1259040460 28800
#      Mon Nov 23 21:27:40 2009 -0800
# Node ID 02d59b67632ff2718c666438979e1785bce5b3d0
# Parent  5d5dc23d6c56a95d6c555f8eb79d798a0240b6be
Stop distribution of conf.texi by autotools

conf.texi must not be distributed.  Instead it must be re-made from
conf.texi.in during the configure stage of compilation.

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,14 @@
+2009-11-23  Rik <octave@nomad.inbox5.com>
+
+	* faq/Makefile.am, interpreter/Makefile.am, liboctave/Makefile.am: 
+	Stop distribution of conf.texi by autotools.  conf.texi must be re-made
+	from conf.texi.in during configure step
+
 2009-11-23  Rik <octave@nomad.inbox5.com>
 
 	* Makefile.am: Add . to SUBDIRS to force automake to compile this
 	directory, and make conf.texi, before recursively compiling SUBDIRS which
 	depend on conf.texi
 
 2009-11-11  Rik <octave@nomad.inbox5.com>
 
diff --git a/doc/faq/Makefile.am b/doc/faq/Makefile.am
--- a/doc/faq/Makefile.am
+++ b/doc/faq/Makefile.am
@@ -22,17 +22,17 @@
 TOPDIR = ../..
 
 include ../../common.mk
 
 TEXINFO_TEX = ../texinfo.tex
 
 info_TEXINFOS = OctaveFAQ.texi
 
-OctaveFAQ_TEXINFOS = ../conf.texi
+nodist_OctaveFAQ_TEXINFOS = ../conf.texi
 
 ../conf.texi:
 	$(MAKE) -C .. conf.texi
 
 all-local: dvi html pdf ps
 
 EXTRA_DIST = OctaveFAQ.dvi OctaveFAQ.html OctaveFAQ.pdf OctaveFAQ.ps
 
diff --git a/doc/interpreter/Makefile.am b/doc/interpreter/Makefile.am
--- a/doc/interpreter/Makefile.am
+++ b/doc/interpreter/Makefile.am
@@ -129,18 +129,20 @@ MUNGED_TEXI_SRC = \
   stmt.texi \
   strings.texi \
   system.texi \
   testfun.texi \
   tips.texi \
   var.texi
 
 info_TEXINFOS = octave.texi
-octave_TEXINFOS = \
-  ../conf.texi \
+nodist_octave_TEXINFOS = \
+  ../conf.texi 
+
+dist_octave_TEXINFOS = \
   contributors.texi \
   $(MUNGED_TEXI_SRC)
 
 TXI_SRC = $(MUNGED_TEXI_SRC:.texi=.txi)
 
 ../conf.texi:
 	$(MAKE) -C .. conf.texi
 
diff --git a/doc/liboctave/Makefile.am b/doc/liboctave/Makefile.am
--- a/doc/liboctave/Makefile.am
+++ b/doc/liboctave/Makefile.am
@@ -28,18 +28,20 @@ AM_MAKEINFOHTMLFLAGS = -I.. -I$(srcdir) 
 
 TEXINFO_TEX = ../texinfo.tex
 
 TEXINPUTS = "..$(PATH_SEPARATOR)$(srcdir)$(PATH_SEPARATOR)$(srcdir)/..$(PATH_SEPARATOR)"
 export TEXINPUTS
 
 info_TEXINFOS = liboctave.texi
 
-liboctave_TEXINFOS = \
-  ../conf.texi \
+nodist_liboctave_TEXINFOS = \
+  ../conf.texi
+
+dist_liboctave_TEXINFOS = \
   array.texi \
   bugs.texi \
   cp-idx.texi \
   dae.texi \
   diffeq.texi \
   error.texi \
   factor.texi \
   fn-idx.texi \
